### DISCUSSION TITLE
How to see the LOG.debug output?

### DISCUSSION THREAD

**Discussion by ucmmesa** [openTCS/opentcs#46] [Q&A] [answered]
Hi,everyone

There are many LOG.debug commands in the project.However,the information from LOG.info could be seen in the console.But where and how can I see the LOLOG.debug output?I use Intellij IDEA as the IDE.

**Comment by swltr** [comment, marked as answer]
openTCS uses slf4j, by default with `java.util.logging`. To configure the latter, you need to modify the file `config/logging.config` in the respective application's directory.
